### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by evans-g-crsj**
closes: #3109

`Test Suites: 27 passed, 27 total
Tests:       155 passed, 155 total
Snapshots:   1 passed, 1 total
Time:        8.361s
Ran all test suites.`

vertical/horizontal align
image

typo
image

bold + align
image

### CODE REVIEW

File: src/pages/variantEntity/TabFrequencies.tsx [hunk 1]
```diff
@@ -108,7 +107,7 @@ const internalColumns = (
   },
   {
     title: 'Frequency',
-    align: AlignmentOptions.center,
+    //TODOalign: AlignmentOptions.center,
```
Comment (francisl) [inline src/pages/variantEntity/TabFrequencies.tsx:110]: please remove todo in code.

File: src/pages/variantEntity/TableSummaryKfStudies.module.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@import 'src/style/themes/default/colors';
+
+.participantNumLink {
+  color: $primary-color;
+  text-decoration: underline;
+  font-weight: bold;
+}
```
Comment (francisl) [inline src/pages/variantEntity/TableSummaryKfStudies.module.scss:7]: can you provide an explanation why this button link is different that the others in the other table and other pages?

Comment (evans-g-crsj) [reply src/pages/variantEntity/TableSummaryKfStudies.module.scss:7]: check in the current code (before the pr) that the style was from a completely different page. In that sense, it's a (small) improvement. 
If I remove that style I do not obtain what is visually desired.

So, in my opinion it s better than what it were prior to the pr. However, it is far from being ok. But we have another ticket that take care of all the links style-wise. At that point, that file could be removed.

Comment (francisl) [reply src/pages/variantEntity/TableSummaryKfStudies.module.scss:7]: But it added technical debt and work to the person who will do the task next. Better to close your PR fast and take the next Issue and fix it correctly.

**Review by francisl — APPROVED** [review]

**Pull request merged**